assets/app/styles/_core.less

```diff
@@ -94,6 +93,20 @@
     }
   }
 }
+
+@media (max-width: @screen-xs-max) {
+  project-filter {
+    width: 100%; // Extends flex container full width of flex parent
+  }
+  .filter .navbar-filter-widget .label-filter {
+    min-width: initial;
+    width: 80%; // Width of the input field that allows for the 'add' button to be appended down to 320px
+  }
+  .filter .navbar-filter-widget .selectize-input {
+    width: auto !important;
+  }
```

Comment:
The initial specificity was because I needed to override https://github.com/kubernetes-ui/label-selector/blob/master/styles/labelFilter.less#L157

I reduced it down to 3 levels but it needs !important to override the one from the label-selector repo